test/server/drain_manager_impl_test.cc

```diff
@@ -17,10 +17,13 @@ namespace Envoy {
 namespace Server {
 namespace {
 
-class DrainManagerImplTest : public testing::Test {
+const int kDrainTimeSeconds(600);
```

Comment:
`constexpr int DrainTickSeconds = 600;`

Envoy naming convention is slightly different from Google's.

Reply:
Done, but can you explain what is meant by "tick" here as compared to time? I understand CPU clocks ticking and stuff, but is it just commonplace to refer to time as ticks?

Reply:
I have no idea why I wrote 'Tick'. I meant DrainTimeSeconds.